src/EnergyPlus/DataConvergParams.hh

```diff
@@ -92,8 +92,8 @@ namespace DataConvergParams {
     constexpr Real64 PlantEnergyToler(10.0); // Tolerance for Energy comparisons (in Watts W)
 
     // Energy Tolerance Calculation, used to scale the answer for a more intuitive answer for comparison
-    constexpr Real64 PlantFlowFlowRateToler(0.01);    // Tolerance for mass flow rate convergence (in kg/s)
-    constexpr Real64 PlantLowFlowRateToler(0.000001); // Tolerance for low flow rate used for determining when plant pumps can be shut down
+    constexpr Real64 PlantFlowFlowRateToler(0.01);   // Tolerance for mass flow rate convergence (in kg/s)
+    constexpr Real64 PlantLowFlowRateToler(1.0E-30); // Tolerance for low mass flow rate used for determining when plant pumps can be shut down
```

Comment:
Absolute tolerances are defined below (perhaps these should be relative to the plant loop flows?):
https://github.com/NREL/EnergyPlus/blob/9d7789a3accef2ed968a6198a90b6218206340e5/src/EnergyPlus/DataHVACGlobals.hh#L72-L79
To avoid further failures, probably the best (and the most consistent with what's already use) is `DataHVACGlobals::VerySmallMassFlow`. It is currently only used for airloops but its name doesn't exclude other fluids.